### ISSUE TITLE
Deactivate epidemics cheat doesn't work

### ISSUE THREAD

**Issue by UncagedFungus** [CorsixTH/CorsixTH#2718]
Has only happened since update to 0.68 on 14th October.
Will click 'Deactivate epidemics', a box pops up telling me so and the receptionist announces that I'm cheating.  But I still get epidemics.
![cheat](https://github.com/user-attachments/assets/c133e2df-191f-40c5-9c8e-8f0718508dd2)

**Comment by lewri** [comment]
I don't believe we currently intend to stop an epidemic already revealed. Unless this occurs after.

**Comment by UncagedFungus** [comment]
It does occur afterwards.

**Comment by lewri** [comment]
Thanks. I've had enough look.

The cheat was added  by #1999 but looks like what was added isn't enough.
https://github.com/CorsixTH/CorsixTH/blob/541e4f933643e8c93191eca34acc6e884e23a272/CorsixTH/Lua/cheats.lua#L138-L147

Currently, that PR stopped any *new* contagious patients happening; but never dealt with contagious patients that already exist.

Moving forward, it would be best to also performing actions against epidemics not yet declared. An epidemic in progress currently likely needs to naturally complete to not cause breakage. The UIInformation text on toggle should then also be updated to something like `Epidemics are disabled. No new epidemics will be created` when disabling it.

**Comment by ARGAMX** [comment]
> https://github.com/CorsixTH/CorsixTH/blob/541e4f933643e8c93191eca34acc6e884e23a272/CorsixTH/Lua/cheats.lua#L145-L145
> 
> Currently, that PR stopped any _new_ contagious patients happening; but never dealt with contagious patients that already exist.
> 

The funniest thing that this `epidemics_disabled` flag is never used outside if this cheat panel...
I have tested this case and discovered that this option does't work at all.
It even doesn't stop any _new_ contagious patients happening.
Then I have checked the PR and came to conclusion that this cheat was never work...
The only thing this option doing is prohibit generating infected patients from this cheat menu... 
So this option not only does not stop new epidemics, it does nothing at all... 😀

> Moving forward, it would be best to also performing actions against epidemics not yet declared. An epidemic in progress currently likely needs to naturally complete to not cause breakage. The UIInformation text on toggle should then also be updated to something like `Epidemics are disabled. No new epidemics will be created` when disabling it.

💯
